core/src/androidMain/kotlin/ManufacturerDataFilter.kt

```diff
@@ -0,0 +1,7 @@
+package com.juul.kable
+
+public class ManufacturerDataFilter(
+    public val manufacturerId: Int,
+    public val manufacturerData: ByteArray,
+    public val manufacturerDataMask: ByteArray?
```

Comment:
Being that this is in the `ManufacturerDataFilter`, having the properties prefixed with `manufacturer` feels a bit redundant to me.

```suggestion
    public val id: Int,
    public val data: ByteArray,
    public val dataMask: ByteArray?
```

Reply:
Agreed.